scripts/actions/mobskills/leaping_cleave.lua

```diff
@@ -20,9 +20,13 @@ end
 mobskillObject.onMobWeaponSkill = function(target, mob, skill)
     local numhits = 1
     local accmod = 3
-    local dmgmod = 4
+    local dmgmod = 2.25
     local info = xi.mobskills.mobPhysicalMove(mob, target, skill, numhits, accmod, dmgmod, xi.mobskills.magicalTpBonus.NO_EFFECT)
     local dmg = xi.mobskills.mobFinalAdjustments(info.dmg, mob, skill, target, xi.attackType.PHYSICAL, xi.damageType.SLASHING, info.hitslanded)
+
+    local duration = xi.mobskills.calculateDuration(mob:getTP(), 15, 30)
```

Comment:
This doesnt have to be in this PR, maybe Xaver knows better than me but I believe the check inside `xi.mobskills.mobStatusEffectMove` should take care of the difference in duration. I always thought it should be the duration is (in this case) 30 and then the resist check will modify that duration based on the resist. Jimmayus sheet says " Stun duration varies with TP(?)" so its not 100% confirmed. My guess would be its just based off the resist state like everything else.

Reply:
Probably the case for enfeeblement mobskills, but for the time being, with the info we have, this is the best course of action.
At least it seems sane that first the TP would affect the base duration and then multipliers, such as resist states, taking effect after.

Reply:
I went and tested on retail. My understanding is that resists can be 1/2 or 1/4 on retail for status effects. I was testing with a 99 job so I suspect I kept resisting but I was getting either 3~ or 7~ second stuns. Since the mob was below 25% they'd use the mobskill at 1000TP. With 15s stun a 1/4 resist would be about 3s and 1/2 would be about 7~. I think the only way to get more accurate with this would be to get some 75 characters that wouldn't resist much and test it a bunch.